.template/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
```

Comment:
I think it's out of date with the newest version in [ios-template](https://github.com/nimblehq/ios-templates/blob/develop/fastlane/Fastfile)

Reply:
@markgravity I don't copy all functions, just to build the app and deploy it to TestFlight first. I'm adding piece by piece so we are able to test if the functions work correctly or not because there might be minor changes when writing Fastlane for Flutter 😁